File: store/dotnetcore.localfile/Models/Question.cs
using System.Collections.Generic;

namespace dotnetcore.localfile.Models
{
    public class Question
    {
        public long? Id { get; set; }
        public string Name { get; set; }
        public string Link { get; set; }
        public string Asked { get; set; }
        public string Viewed { get; set; }
        public long UpVote { get; set; }
        public long Favorite { get; set; }
        public string PostContent { get; set; }
        public IEnumerable<string> Tags { get; set; }
    }
}

File: parser/java.parser/src/main/java/net/alanwei/robots/models/Question.java
package net.alanwei.robots.models;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class Question {
    private Long id;
    private String name;
    private String link;
    private String asked;
    private String viewed;
    private Long upVote;
    private Long favorite;
    private String postContent;
    private List<String> tags;
}
